libraries/botbuilder-ai-orchestrator/src/index.ts

```diff
@@ -6,5 +6,5 @@
  * Licensed under the MIT License.
  */
 
-export { OrchestratorRecognizer, LabelType } from './orchestratorRecognizer';
-export { OrchestratorComponentRegistration } from './orchestratorComponentRegistration';
+export * from './orchestratorRecognizer';
+export * from './orchestratorBotComponent';
```

Comment:
Can you roll this back? I'm not a fan of the" `export * from` in `src/index.ts`" pattern as we've been burned in the past by contributors exporting code that should have been internal according to the .NET implementation. ([botbuilder-dialogs-adaptive example](https://github.com/microsoft/botbuilder-js/pull/2171/files#diff-18b57acf181539c55109c3a13dfb10426a10813af0cbfc21a1dde04a2c4a4fce))

The implicit pattern we've followed is that if isn't exported from `src/index.ts`, it's "internal".

It goes without saying that while TypeScript and Python doesn't have a good `internal` concept for sharing code across files and packages, it would be detrimental to prevent the .NET SDK from using this idiomatic feature.